README.md

```diff
@@ -1,151 +1,72 @@
-  a "child" hosted zone (e.g. `myclusters.mydomain.com`) if you want to isolate them.  Note that with AWS Route53,
-  you can have subdomains in a single hosted zone, so you can have `cluster1.testclusters.mydomain.com` under
-  `mydomain.com`.
+We like to think of it as `kubectl` for clusters. 
 
-* Pick a DNS name under this zone to be the name of your cluster.  kops will set up DNS so your cluster
-  can be reached on this name.  For example, if your zone was `mydomain.com`, a good name would be
-  `kubernetes.mydomain.com`, or `dev.k8s.mydomain.com`, or even `dev.k8s.myproject.mydomain.com`. We'll call this `NAME`.
+The tool enables the user to deploy functional Kubernetes clusters in the cloud ([AWS](/docs/aws.md) or GCE). 
 
-* Set `AWS_PROFILE` (if you need to select a profile for the AWS CLI to work)
+We encourage everyone to read more about [how it works](/docs/how_it_works.md) or check out some of the [other interesting modes](/docs/commands.md#other-interesting-modes).
 
-* Pick an S3 bucket that you'll use to store your cluster configuration - this is called your state store.  You
-  can `export KOPS_STATE_STORE=s3://<mystatestorebucket>` and then kops will use this location by default.  We
-  suggest putting this in your bash profile or similar.  A single registry can hold multiple clusters, and it
-  can also be shared amongst your ops team (which is much easier than passing around kubecfg files!)
-
-* Run "kops create cluster" to create your cluster configuration:
-```
-${GOPATH}/bin/kops create cluster --cloud=aws --zones=us-east-1c ${NAME}
-```
-(protip: the --cloud=aws argument is optional if the cloud can be inferred from the zones)
-
-* Run "kops update cluster" to build your cluster:
-```
-${GOPATH}/bin/kops update cluster ${NAME} --yes
-```
+### Features 
 
-If you have problems, please set `--v=8` and open an issue, and ping justinsb on slack!
-
-## Create kubecfg settings for kubectl
-
-(This step is actually optional; `update cluster` will do it automatically after cluster creation.
- But we expect that if you're part of a team you might share the KOPS_STATE_STORE, and then you can do
- this on different machines instead of having to share kubecfg files)
-
-To create the kubecfg configuration settings for use with kubectl:
-
-```
-export KOPS_STATE_STORE=s3://<somes3bucket>
-# NAME=<kubernetes.mydomain.com>
-${GOPATH}/bin/kops export kubecfg ${NAME}
-```
-
-You can now use kubernetes using the kubectl tool (after allowing a few minutes for the cluster to come up):
-
-```kubectl get nodes```
+* HA (Highly Available) Kubernetes clusters
+* Automated Kubernetes cluster [CRUD](/docs/commands.md) for the cloud ([AWS](/docs/aws.md) and GCE)
+* Uses a state-sync model for **dry-run** and automatic **idempotency**
+* Custom support for `kubectl` [add-ons](docs/addons.md)
+* Can produce configurations in other formats 
+    * [Terraform configuration](/docs/terraform.md)
+    * Cloud-Init configuration
```

Comment:
FYI, I'm thinking about removing cloud-init support also as it doesn't seem anyone cares to use it.